thirdeye-ui/src/app/pages/welcome-page/create-datasource/onboard-datasets/onboard-datasets-page.component.tsx

```diff
@@ -228,66 +233,75 @@ export const WelcomeSelectDatasets: FunctionComponent = () => {
                                     <EmptyStateSwitch
                                         emptyState={
                                             <Box p={5}>
-                                                {t(
-                                                    "message.no-datasets-available"
-                                                )}
+                                                <Alert
+                                                    icon={<InfoOutlined />}
+                                                    severity="info"
+                                                    variant="outlined"
+                                                >
+                                                    {t(
+                                                        "message.no-datasets-available-in-datasource"
+                                                    )}
+                                                </Alert>
                                             </Box>
                                         }
                                         isEmpty={tables?.length === 0}
                                     >
-                                        <Box
-                                            alignItems="flexStart"
-                                            data-testid={
-                                                ONBOARD_DATASETS_TEST_IDS.DATASETS_OPTIONS_CONTAINER
-                                            }
-                                            display="flex"
-                                            id="datasets-options-container"
-                                            mt={2}
-                                        >
-                                            <FormGroup>
-                                                <Button
-                                                    color="primary"
-                                                    variant="text"
-                                                    onClick={() => {
-                                                        tables &&
-                                                            setSelectedDatasets(
-                                                                tables.map(
-                                                                    ({
-                                                                        name,
-                                                                    }) => name
-                                                                )
-                                                            );
-                                                    }}
-                                                >
-                                                    {t("label.select-all")}
-                                                </Button>
-                                                <Divider />
+                                        {!isEmpty(tables) && (
```

Comment:
nit: Is this check needed ? From the component `EmptyStateSwitch` name it seems the children will be rendered if only `isEmpty` prop is false ?

Reply:
Ahhh good catch. Removed it. Thanks!